Rocket.Chat/Managers/Launcher/BugTrackingCoordinator.swift

```diff
@@ -14,8 +14,29 @@ import Crashlytics
 import Instabug
 #endif
 
+let kCrashReportingDisabledKey = "kCrashReportingDisabledKey"
+
 struct BugTrackingCoordinator: LauncherProtocol {
+
+    static var isCrashReportingDisabled: Bool {
+        return UserDefaults.standard.bool(forKey: kCrashReportingDisabledKey)
+    }
+
+    static func toggleCrashReporting(disabled: Bool) {
+        UserDefaults.standard.set(disabled, forKey: kCrashReportingDisabledKey)
+
+        if disabled {
+            anonymizeCrashReports()
```

Comment:
@rafaelks if for `disabled` we just anonymize the crash reports, why not always launch on `prepareToLaunch` and just anonymize/identify on toggling?